backend/code_review_backend/issues/management/commands/cleanup_issues.py

```diff
@@ -29,24 +32,51 @@ def add_arguments(self, parser):
 
     def handle(self, *args, **options):
         clean_until = timezone.now() - timedelta(days=options["nb_days"])
+
         rev_to_delete = Revision.objects.filter(
             base_repository__slug__in=["autoland", "mozilla-central"],
             head_repository__slug__in=["autoland", "mozilla-central"],
             created__lte=clean_until,
         )
+        count_rev = rev_to_delete.count()
 
-        count = rev_to_delete.count()
-        if not count:
+        if not count_rev:
             logger.info("Didn't find any old revision to delete.")
             return
 
-        # Delete revisions, with their related IssueLink and Diff
         logger.info(
-            f"Retrieved {count} old revisions from either autoland or mozilla-central to be deleted."
+            f"Retrieved {count_rev} old revisions from either autoland or mozilla-central to be deleted."
         )
-        _, stats = rev_to_delete.delete()
-        # Delete issues that are not linked to a revision anymore
-        _, issues_stats = Issue.objects.filter(revisions__isnull=True).delete()
-        stats.update(issues_stats)
+
+        stats = {}
+
+        iterations = math.ceil(count_rev / DEL_CHUNK_SIZE)
+        i = 0
+        for start in range(0, count_rev, DEL_CHUNK_SIZE):
```

Comment:
@abpostelnicu you have a very useful [enumerate](https://docs.python.org/3/library/functions.html#enumerate) function to do that in Python:
```suggestion
        for i, start in enumerate(range(0, count_rev, DEL_CHUNK_SIZE), start=1):
```
`start=1` allows to start from index 1 at the first iteration (instead of zero)